variables.tf

```diff
@@ -13,6 +13,12 @@ variable "app_name" {
   type = string
 }
 
+variable "aws_region" {
+  description = "AWS region to run in (e.g., us-east-1)"
+  default     = ""
```

Comment:
Why not `"us-east-1"`? I _think_ having this as the default could cause an error if it's not specified.

Reply:
I think I want it to error if it isn't specified.  The user of the module should specify what region he wants it to run in.

Reply:
Ok, your preference, but personally I would remove the default for that.